src/keri/app/cli/commands/multisig/join.py

```diff
@@ -196,7 +201,12 @@ def incept(self, attrs):
 
         if approve:
             if self.auto:
-                alias = "test alias"
+                alias = self.alias
+            elif self.alias:
```

Comment:
self.alias should only be used if self.auto is True.

Reply:
This was changed in the most recent commit to have a nested if statement checking of the alias (now "group") is set.
For example:
```python
            if self.auto:
                if self.group is None:
                    group = "default-group"
                else:
                    group = self.group
```